bindings_ffi/src/mls.rs

```diff
@@ -986,24 +986,34 @@ impl From<VerifiedKeyPackageV2> for FfiKeyPackageStatus {
 
 impl From<AssociationState> for FfiInboxState {
     fn from(state: AssociationState) -> Self {
+        let mut sorted_members: Vec<_> = state.members().values().cloned().collect();
```

Comment:
Why not sort this at xmtp_id/xmtp_mls level so we don’t have to maintain the same sorting code across all our bindings?